bundle-workflow/tests/test_integ_workflow/integ_test/test_local_test_cluster.py

```diff
@@ -205,27 +204,26 @@ def test_terminate_process(
     def test_terminate_process_timeout(
         self, mock_logging, mock_terminate, mock_wait, mock_process
     ):
-        with open("stdout-tmp.txt", "w") as self.local_test_cluster.stdout, open(
-            "stderr-tmp.txt", "w"
-        ) as self.local_test_cluster.stderr:
-            mock_wait.side_effect = subprocess.TimeoutExpired(cmd="pass", timeout=1)
-            with self.assertRaises(subprocess.TimeoutExpired):
-                self.local_test_cluster.process = self.process
-                self.local_test_cluster.terminate_process()
-                mock_process.assert_called_once_with(self.process.pid)
-                mock_terminate.assert_called_once()
-                mock_wait.assert_called_with(10)
-                mock_logging.info.assert_has_calls(
-                    [
-                        call(f"Sending SIGTERM to PID {self.process.pid}"),
-                        call("Waiting for process to terminate"),
-                        call(
-                            "Process did not terminate after 10 seconds. Sending SIGKILL"
-                        ),
-                        call("Waiting for process to terminate"),
-                        call("Process failed to terminate even after SIGKILL"),
-                        call(
-                            f"Process terminated with exit code {self.process.returncode}"
-                        ),
-                    ]
-                )
+        self.local_test_cluster.stdout = tempfile.NamedTemporaryFile(dir=self.local_test_cluster.work_dir)
+        self.local_test_cluster.stderr = tempfile.NamedTemporaryFile(dir=self.local_test_cluster.work_dir)
+        mock_wait.side_effect = subprocess.TimeoutExpired(cmd="pass", timeout=1)
+        with self.assertRaises(subprocess.TimeoutExpired):
+            self.local_test_cluster.process = self.process
+            self.local_test_cluster.terminate_process()
+            mock_process.assert_called_once_with(self.process.pid)
```

Comment:
I think this is wrongly indented, and the code is never reached because an exception is raised from `terminate_process()` and caught by `assertRaises`.

Reply:
Ah my bad! Will fix the indentation.